doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -91,6 +105,20 @@ associativity rules have to be given.
    time of the declaration of the notation. The right-hand side is
    currently typed only at use time but this may change in the future.
 
+.. exn:: Unterminated string in notation
+
+   Occurs when the notation string contains an unterminated quoted
+   string, as e.g. in :g:`Notation "A ""an unended string B"`. What
+   would be correct is typically :g:`Notation "A ""an ended string"" B`.
```

Comment:
Would it be ok to use "may" rather than "probably", here and below?

Do the examples really help actually? They somehow look a bit artificial (and complicated).

Reply:
"May", "probably", either one is fine.  But I agree the examples are not so helpful.

Reply:
I'd suggest using `Reserved Notation` instead of `:= "x"`, to make the examples slightly less artificial

Reply:
Using `Reserved` adopted.